node/cmd/guardiand/node.go

```diff
@@ -491,19 +489,12 @@ func runNode(cmd *cobra.Command, args []string) {
 	if *terraContract == "" {
 		logger.Fatal("Please specify --terraContract")
 	}
-
-	if *unsafeDevMode {
-		if *algorandRPC == "" {
-			logger.Fatal("Please specify --algorandRPC")
-		}
-		if *algorandToken == "" {
-			logger.Fatal("Please specify --algorandToken")
-		}
-		if *algorandContract == "" {
-			logger.Fatal("Please specify --algorandContract")
-		}
+	if *algorandIndexerRPC == "" {
+		logger.Fatal("Please specify --algorandIndexerRPC")
+	}
+	if *algorandAppID == 0 {
```

Comment:
Do you want to check for algorandIndexerToken as well?

Reply:
added